extensions/sql-migration/src/wizard/databaseBackupPage.ts

```diff
@@ -1820,22 +1819,22 @@ export class DatabaseBackupPage extends MigrationWizardPage {
 	}
 
 	private async loadBackupButton(dbIndex: number, isBackupInProgress: boolean, cancelling: boolean) {
-		const backupTriggered = this.migrationStateModel._databaseBackup.backupTasks[dbIndex] && this.migrationStateModel._databaseBackup.backupTasks[dbIndex].status === 0;
+		const backupTriggered = this.migrationStateModel._databaseBackup.backupTasks[dbIndex] && this.migrationStateModel._databaseBackup.backupTasks[dbIndex].status === backupStatus.InProgress;
 		await this._blobContainerAction[dbIndex].updateProperty('enabled', this.migrationStateModel._databaseBackup.blobs[dbIndex].blobContainer && !cancelling);
 		await this._networkShareAction[dbIndex].updateProperty('enabled', this._networkShareLocations[dbIndex].valid && this._networkShareLocations[dbIndex].value && this._networkShareLocations[dbIndex].value?.trim() !== '' && !cancelling);
 		await this._blobContainerAction[dbIndex].updateProperty('iconPath', !backupTriggered ? IconPathHelper.backup : IconPathHelper.cancel);
 		await this._networkShareAction[dbIndex].updateProperty('iconPath', !backupTriggered ? IconPathHelper.backup : IconPathHelper.cancel);
 		if (this.migrationStateModel._databaseBackup.backupTasks[dbIndex]) {
 			switch (this.migrationStateModel._databaseBackup.backupTasks[dbIndex].status) {
-				case 1:
-				case 2: {
+				case backupStatus.Canceled:
+				case backupStatus.Successed: {
 					await this._networkShareBackupLoading[dbIndex].updateProperty('value', '');
 					await this._blobContainerBackupLoading[dbIndex].updateProperty('value', '');
 					break;
 				}
-				case 3: {
-					await this._networkShareBackupLoading[dbIndex].updateProperty('value', 'failed');
-					await this._blobContainerBackupLoading[dbIndex].updateProperty('value', 'failed');
+				case backupStatus.Failed: {
+					await this._networkShareBackupLoading[dbIndex].updateProperty('value', constants.StatusLookup['failed']);
+					await this._blobContainerBackupLoading[dbIndex].updateProperty('value', constants.StatusLookup['failed']);
```

Comment:
can 'failed' be replaced with the constant 'backupStatus.Failed'

Reply:
backupStatus.Failed is a number type